src/widgets/AppWindow.js

```diff
@@ -667,7 +673,6 @@ export default class AppWindow extends createjs.Container {
     commentArea.y = 140;
     form.addChild(commentArea);
     form.accessible.addChild(commentArea);
-
```

Comment:
Nit: not seeing a reason to delete this blank line that is providing separation between different chunks of code.

Reply:
updated